models/users.js

```diff
@@ -576,6 +576,52 @@ const fetchAllUsers = async () => {
   return users;
 };
 
+const fetchUsersWithToken = async () => {
+  try {
+    const users = [];
+    const usersRef = await userModel.where("tokens", "!=", false).get();
+    usersRef.forEach((user) => {
+      users.push(userModel.doc(user.id));
+    });
+    return users;
+  } catch (err) {
+    logger.error(`Error while fetching all users with tokens field: ${err}`);
+    throw err;
+  }
+};
+
+const removeGitHubToken = async (users) => {
+  try {
+    const length = users.length;
+
+    let numberOfBatches = length / 500;
+    numberOfBatches = numberOfBatches + 1;
+
+    const batchArray = [];
+    for (let i = 0; i < numberOfBatches; i++) {
+      const batch = firestore.batch();
+      batchArray.push(batch);
+    }
+
+    let batchIndex = 0;
+    let operations = 0;
```

Comment:
Tejas has strictly mentioned that No PRs with `Injection Sink` warning will be merged  👀

Reply:
@heyrandhir I was able to work around with the help of @TanishqSingla

Reply:
`somehow when we're accessing an object using a key, it takes it any type instead of number, and maybe that is causing it to throw this warning as the eslint doens't know if the key is a number or not` according to @TanishqSingla this is the problem